### PR TITLE
Added Automatic-Module-Name to manifest

### PR DESCRIPTION
**Pull request by scholzi100**
This PR added the Automatic-Module-Name ("redis.clients.jedis") to the projects manifest.

This PR is related to Issue #2044. 

Compile dependencies supporting Java Module System:
- SLF4J - yes
- commons - yes [Issue 33](https://github.com/apache/commons-pool/pull/31 )

### CODE REVIEW

**Comment by scholzi100** [comment]
@sazzad16 can you please review this PR. :grimacing:

**Comment by wangminpku** [comment]
Dear  scholzi100 ( @scholzi100) and Jedis team  (@xetorthio), (@HeartSaVioR), (@sazzad16 )
We are working on automatically identifying  and clustering the related pull requests for code review. We have found that [PR#1909](https://github.com/xetorthio/jedis/pull/1909) might be related to this one, because both of them modified pom.xml in the same section. 
We would really appreciate it if you could help us to validate if the information from [PR#1909](https://github.com/xetorthio/jedis/pull/1909) is helpful when you review the current PR. 
Thank you very much for your time!

**Comment by scholzi100** [comment]
@wangminpku PR#1909 does not seam to be related to this PR.

**Comment by gkorland** [comment]
@scholzi100 is there a way to test it, to verify the build process?

**Comment by scholzi100** [comment]
@gkorland To verify the module is working. I think a seperate module using java 11 with module-info would be required.

**Comment by sazzad16** [comment]
@scholzi100 Would you be able to add the module name `redis.clients.jedis` as property and use that property in `Automatic-Module-Name` tag? Thanks!

**Review by sazzad16 — CHANGES_REQUESTED** [review]
property tags don't match

File: pom.xml [hunk 1]
```diff
@@ -50,6 +50,7 @@
 		<cluster-hosts>localhost:7379,localhost:7380,localhost:7381,localhost:7382,localhost:7383,localhost:7384,localhost:7385</cluster-hosts>
 		<github.global.server>github</github.global.server>
 		<log4j.version>2.13.3</log4j.version>
+		<module.name>redis.clients.jedis</module.name>
```
Comment (sazzad16) [inline pom.xml:53]:
```suggestion
		<jedis.module.name>redis.clients.jedis</jedis.module.name>
```

File: pom.xml [hunk 2]
```diff
@@ -200,9 +201,12 @@
 				<artifactId>maven-jar-plugin</artifactId>
 				<version>3.0.2</version>
 				<configuration>
-					<archive>  
+					<archive>
 						<manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
-					</archive> 
+						<manifestEntries>
+							<Automatic-Module-Name>${module.name}</Automatic-Module-Name>
```
Comment (sazzad16) [inline pom.xml:207]:
```suggestion
							<Automatic-Module-Name>${jedis.module.name}</Automatic-Module-Name>
```

**Review by sazzad16 — CHANGES_REQUESTED** [review]

**Review by sazzad16 — APPROVED** [review]

**Review by gkorland — APPROVED** [review]

**Pull request merged**